AK/String.h

```diff
@@ -160,6 +160,7 @@ class String {
 
     [[nodiscard]] bool is_null() const { return !m_impl; }
     [[nodiscard]] ALWAYS_INLINE bool is_empty() const { return length() == 0; }
+    [[nodiscard]] ALWAYS_INLINE bool is_null_or_empty() const { return !m_impl || length() == 0; }
```

Comment:
This is equivalent to `String::Is_empty()` (since `length()` null-checks `m_impl`)